### ISSUE TITLE
Feature: OS Function to read lines until null from STDIN

### ISSUE THREAD

**Issue by AtjonTV** [vlang/v#738]
In os we have a function called `get_line`. It allows to read 1 line from stdin.

Making a wrapper for that function, we can make a `get_lines` that returns all input from stdin until a null input (string length == 0).

Here is my version of this function that collects all input in one string:

```go
pub fn get_lines() string {
        mut line := ''
        mut inputstr := ''
        for {
                line = get_line()
                if(line.len <= 0) {
                        break
                }
                line = line.trim_space()
                inputstr += line
        }
        return inputstr
}
```
(I just pasted that under `get_line` in `os/os.v`)

Anothing solution can be to return a string array with the lines:

```go
pub fn get_lines() []string {
        mut line := ''
        mut inputstr := []string
        for {
                line = get_line()
                if(line.len <= 0) {
                        break
                }
                line = line.trim_space()
                inputstr << line
        }
        return inputstr
}
```

**Comment by AtjonTV** [comment]
To be able to use both, I simply renamed the first one to `get_lines_joined`.

With

* `get_line` I read 1 line
* `get_lines` I read all lines into a array until a null input
* `get_lines_joined` I read all lines into a string until a null input

**Comment by be1** [comment]
Mmh... why using trim_space() ?

I think `get_lines` should either just get raw lines or be documented that they trim space in [doc](https://modules.vlang.io/os.html#get_lines). Maybe a bool trim parameter to the function would be useful ?

**Comment by JalonSolov** [comment]
A "line" is basically defined as "a sequence of characters ending with a linefeed".  So if you 
"get a line" your should expect a linefeed (or carriage-return + linefeed) at the end.

`trim_space()` is called to remove it when you don't want it.

**Comment by be1** [comment]
Actually an issue is that `trim_space()` seems to remove not only linefeed, but also tabs and spaces at the beginning of the line (and the end too), so that `get_lines()` (which trims spaces) does not return the exact lines when there are spaces or tabs in the original line.

**Comment by JalonSolov** [comment]
Most of the time, that's what you want.  If you don't, there are other `trim` routines listed on https://modules.vlang.io

**Comment by AtjonTV** [comment]
@be1 

"why using trim_space() ?"
Well, the reason is stupid.

At the time I wrote an implementation of a ini-like configuration file format in V.
Because V was so new, it didn't have a "read all lines until you get an empty line" function.
So I simply wrote a function that did what I needed, which included ignoring leading and trailing white spaces.
Later I opened an issue and, as I already had the code, also a pull request.
The pr was merged without it being a concern.
